Q000 Double/Single quotes message & config

Hello,

Could the readme be updated to show how to configure Q000?

Currently, the readme says the message is "Single quotes found but double quotes preferred". However, the default message seems to be "Double quotes found but single quotes preferred", and I can't figure out how to configure this.

```toml
# pyproject.toml
[tool.ruff]
line-length=88

[tool.ruff.flake8-quotes]
# configured based on example in readme, but the inline option doesn't seem to do anything
docstring-quotes = "double"
inline-quotes = "double"
```

```py
# test.py
ABC = "str"
```

```shell
# CLI
root@5d53afc0df87:/home# ruff --select Q .
Found 1 error(s).
test.py:1:7: Q000 Double quotes found but single quotes preferred
```

Just a little blurb in the readme in the `flake8-quotes` section about how to configure it would be nice. (I'd personally vouch for defaulting to prefer double quotes to match black, even though this doesn't match with the original flake-quotes)

--- a/src/flake8_quotes/settings.rs
+++ b/src/flake8_quotes/settings.rs
@@ -2,7 +2,7 @@
 
 use serde::{Deserialize, Serialize};
 
-#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize, Hash)]
 #[serde(deny_unknown_fields, rename_all = "kebab-case")]
 pub enum Quote {
     Single,
@@ -18,7 +18,7 @@ pub struct Options {
     pub avoid_escape: Option<bool>,
 }
 
-#[derive(Debug)]
+#[derive(Debug, Hash)]
 pub struct Settings {
     pub inline_quotes: Quote,
     pub multiline_quotes: Quote,

--- a/src/pep8_naming/settings.rs
+++ b/src/pep8_naming/settings.rs
@@ -29,7 +29,7 @@ pub struct Options {
     pub staticmethod_decorators: Option<Vec<String>>,
 }
 
-#[derive(Debug)]
+#[derive(Debug, Hash)]
 pub struct Settings {
     pub ignore_names: Vec<String>,
     pub classmethod_decorators: Vec<String>,

--- a/src/settings/mod.rs
+++ b/src/settings/mod.rs
@@ -84,14 +84,19 @@ impl Settings {
 
 impl Hash for Settings {
     fn hash<H: Hasher>(&self, state: &mut H) {
-        self.line_length.hash(state);
+        // Add base properties in alphabetical order.
         self.dummy_variable_rgx.as_str().hash(state);
         for value in self.enabled.iter() {
             value.hash(state);
         }
+        self.line_length.hash(state);
         for value in self.per_file_ignores.iter() {
             value.hash(state);
         }
+        self.target_version.hash(state);
+        // Add plugin properties in alphabetical order.
+        self.flake8_quotes.hash(state);
+        self.pep8_naming.hash(state);
     }
 }
 

--- a/src/settings/types.rs
+++ b/src/settings/types.rs
@@ -11,7 +11,7 @@ use crate::checks::CheckCode;
 use crate::checks_gen::CheckCodePrefix;
 use crate::fs;
 
-#[derive(Clone, Debug, PartialOrd, PartialEq, Eq, Serialize, Deserialize)]
+#[derive(Clone, Debug, PartialOrd, PartialEq, Eq, Serialize, Deserialize, Hash)]
 pub enum PythonVersion {
     Py33,
     Py34,

